public/locales/pl/common.json

```diff
@@ -9,6 +9,16 @@
         "surnamePlaceholder": "Grabowska",
         "emailPlaceholder": "kontakt@sanah.pl"
     },
+    "loginForm": {
+        "title": "Zaloguj się",
+        "emailInptLabel": "Adres E-mail",
+        "emailInputPlaceholder": "kontakt@sanah.pl",
```

Comment:
don't repeat translations keys - we're using i18next to not repeat these strings